src/DatePicker.elm

```diff
@@ -456,7 +458,10 @@ view pickedDate settings (DatePicker ({ open } as model)) =
                     |> value
                 ]
     in
-        div [ class "container" ]
+        div
+            [ class "container"
+            , Attrs.classList settings.containerClassList
```

Comment:
Nice! Question (I don't know off the top of my head): Do `class` and `classList` just get implicitly concatenated in some way? Ie they don't fight/have undefined behavior?

Reply:
(After checking the docs) To be safe, since they don't seem to promise anything, maybe we just `("container", True) :: settings.containerClassList`?

Reply:
yeah mate, totally agreed, I've changed the code, what do you think about now @bbqbaron ? 💃